lib/plugins/aws/provider/awsProvider.js

```diff
@@ -237,6 +237,56 @@ class AwsProvider {
             },
           },
         },
+        resources: {
+          properties: {
+            // Not replicating the full JSON schema from https://s3.amazonaws.com/cfn-resource-specifications-us-east-1-prod/schemas/2.15.0/all-spec.json
+            // as that gets into the specifics for each resource type.
+            //
+            // The only required attribute is `Type`; `Properties` and other common attributes are optional.
+            // See also https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resources-section-structure.html
+            Resources: {
+              type: 'object',
+              patternProperties: {
+                '^[a-zA-Z0-9]{1,255}$': {
+                  type: 'object',
+                  properties: {
+                    Type: { type: 'string' },
+                  },
+                  required: ['Type'],
```

Comment:
Currently `resources.Resources` serve as an _extension_ object, through which resources as created by the Framework can be extended, in such cases users do not provide `Type`, and we should not require it